ide/xml.text/src/org/netbeans/modules/xml/text/completion/ElementResultItem.java

```diff
@@ -123,6 +123,7 @@ public boolean substituteText( JTextComponent c, int offset, int len, int modifi
      */
 //    static class EndTag extends ElementResultItem {
 //    }
-    
+
+    @Deprecated
     Color getPaintColor() { return LFCustoms.shiftColor(COLOR); }
```

Comment:
This is wrong. The @deprecated javadoc annotation was on the `EndTag` class, nut on getPaintColor. The author commenting out the class should have removed it.